src/components/Radios/Radios.tsx

```diff
@@ -22,13 +22,20 @@ const Radios: React.FC<RadiosProps> = ({
     setWhichIsChecked,
     title,
     validation,
+    selectedState,
 }) => {
-    const formattedID = id.replace(/\s/g, '-');
+    const formattedID = id?.replace(/\s/g, '-');
     const isCreditCards = Boolean(typeof items[0] === 'object');
     const [whichIsChecked, setChecked] = useState<number>(
         setWhichIsChecked || items.length,
     );
 
+    useEffect(() => {
+        if (typeof selectedState != 'undefined') {
```

Comment:
Nice solution. This helps to avoid problems with the `0` number

Reply:
Since we already have `setWhichIsChecked` we need to replace `selectedState` to `setWhichIsChecked` here

Reply:
Discussed on Slack, this is a breaking a change, merging current solution